src/sgame/botlib/bot_load.cpp

```diff
@@ -75,12 +75,12 @@ void BotAssertionInit()
 
 void BotSaveOffMeshConnections( NavData_t *nav )
 {
-	char filePath[ MAX_QPATH ];
 	fileHandle_t f = 0;
 
 	std::string mapname = Cvar::GetValue( "mapname" );
-	Com_sprintf( filePath, sizeof( filePath ), "maps/%s-%s.navcon", mapname.c_str(), nav->name );
-	trap_FS_FOpenFile( filePath, &f, fsMode_t::FS_WRITE );
+	std::string filePath = Str::Format(
+		"maps/%s-%s.navcon", mapname, BG_Class( nav->species )->name );
```

Comment:
Totally bikeshedding, but I believe having `"maps/%s-%s.navcon"` on the same line as `Str::Format(` would make it more readable.

Reply:
I believe it's better if at least the first argument is on the same line, except maybe specific rare situations where a bunch of arguments are all on their own lines, aligned.

In the same way, this looks wonky to me:

```
		Log::Notice(
			"Could not allocate Detour Navigation Mesh Query for navmesh %s", speciesName );
```

Reply:
Done